tsconfig.json

```diff
@@ -6,15 +6,12 @@
     "preserveConstEnums": true,
     "moduleResolution": "node",
     "experimentalDecorators": true,
-    "noImplicitAny": true,
     "noEmitOnError": true,
     "jsx": "react",
     "outDir": "compiled/src",
-    "strictNullChecks": true,
     "noImplicitThis": true,
-    "inlineSourceMap": true,
-    "noUnusedLocals": true,
-    "noUnusedParameters": true
```

Comment:
These are disabled checks I was talking about in the PR description; *⚠ we need to put them all back*.